test/battle/ability/parental_bond.c

```diff
@@ -24,23 +24,166 @@ SINGLE_BATTLE_TEST("Parental Bond converts Tackle into a two-strike move")
     }
 }
 
+SINGLE_BATTLE_TEST("Parental Bond does not convert Triple Kick into a two-strike move")
+{
+    GIVEN {
+        ASSUME(gBattleMoves[MOVE_TRIPLE_KICK].split != SPLIT_STATUS);
+        ASSUME(gBattleMoves[MOVE_TRIPLE_KICK].strikeCount == 3);
+        ASSUME(gBattleMoves[MOVE_TRIPLE_KICK].effect == EFFECT_TRIPLE_KICK);
```

Comment:
Is the assume for the effect needed here it shouldn't effect the test in any way?